newIDE/app/src/Export/ExportDialog/index.js

```diff
@@ -0,0 +1,198 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import * as React from 'react';
+import Dialog from '../../UI/Dialog';
+import HelpButton from '../../UI/HelpButton';
+import FlatButton from '../../UI/FlatButton';
+import BuildsDialog from '../Builds/BuildsDialog';
+import AuthenticatedUserContext from '../../Profile/AuthenticatedUserContext';
+import ExportLauncher from './ExportLauncher';
+import { type ExportPipeline } from '../ExportPipeline.flow';
+import { useOnlineStatus } from '../../Utils/OnlineStatus';
+import AlertMessage from '../../UI/AlertMessage';
+import { Tab, Tabs } from '../../UI/Tabs';
+import ExportHome from './ExportHome';
+
+const styles = {
+  icon: { width: 40, height: 40 },
+  disabledItem: { opacity: 0.6 },
+  content: { padding: 8 },
+};
+
+export type ExporterSection = 'assisted' | 'manual' | '';
```

Comment:
Is the value `''` useful? It may look like a hack to not handle a possibly null value in a component state.